modules/overrides/limits.go

```diff
@@ -6,11 +6,15 @@ import (
 )
 
 const (
-	// Local ingestion rate strategy
+	// LocalIngestionRateStrategy indicates that this limit can be evaluated in local terms only
 	LocalIngestionRateStrategy = "local"
-
-	// Global ingestion rate strategy
+	// GlobalIngestionRateStrategy indicates that an attempt should be made to consider this limit across the entire Tempo cluster
 	GlobalIngestionRateStrategy = "global"
+
+	// ErrorPrefixTooManyTraces is used to flag batches from the ingester that were rejected b/c they had too many traces
+	ErrorPrefixTooManyTraces = "TOO_MANY_TRACES:"
```

Comment:
`too_many_traces` could also be interpreted as a rate limit. Thoughts on tweaking it or an alternate string like `live_traces_exceeded` ?

Reply:
agree :+1: